langchain/vectorstores/pgvector.py

```diff
@@ -19,7 +20,7 @@
 Base = declarative_base()  # type: Any
 
 
-ADA_TOKEN_COUNT = 1536
+PGVECTOR_VECTOR_SIZE = int(os.getenv("PGVECTOR_VECTOR_SIZE", default="1536"))
```

Comment:
does default value have to be a string (vs just being int)?

Reply:
My idea of having it as a string here is because env variables are also all incomming as strings so having the default as string would make it clear why i added the int conversion.
I think the int conversion ist not 100% necessary for the pgvector library but it will error early when you put a non-number in to the environment variable